### PR TITLE
Sync docs with Memcached stubs

### PR DESCRIPTION
**Pull request by Girgias**
See: https://github.com/php-memcached-dev/php-memcached

### CODE REVIEW

**Comment by afilina** [comment]
@remicollet Gentle reminder to review the changes.

**Comment by markdavidsonEE** [comment]
Would be great to see this fixed!

**Comment by afilina** [comment]
@cmb69 Can this PR be considered invalid, or is it fixable?

**Comment by Girgias** [comment]
I just need to re-run gen_stubs on the latest Memcached stubs

File: reference/memcached/memcached.xml [hunk 1]
```diff
@@ -22,20 +21,21 @@
...
     <classsynopsisinfo>
      <ooclass>
       <classname>Memcached</classname>
      </ooclass>
     </classsynopsisinfo>
-<!-- }}} -->
 
-    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.memcached')/db:refentry/db:refsect1[@role='description']/descendant::db:constructorsynopsis[1])">
+    <classsynopsisinfo role="comment">&Methods;</classsynopsisinfo>
+    <xi:include xpointer="xmlns(db=http://docbook.org/ns/docbook) xpointer(id('class.memcached')/db:refentry/db:refsect1[@role='description']/descendant::db:constructorsynopsis[not(@role='procedural')])">
```
Comment (kocsismate) [inline reference/memcached/memcached.xml:35]: The role should be the class name everywhere. Are you using an older gen_stub.php version?

Comment (Girgias) [reply reference/memcached/memcached.xml:35]: Don't think so it's the latest gen_stub, but I didn't rerun it on the class synopsis which is probably the issue. Let me fix that.

**Comment by afilina** [comment]
@kocsismate Gentle nudge since this PR is awaiting your review.

**Review by kocsismate — APPROVED** [review]

**Pull request merged**